tools/jvm/src/main/scala/org/scalajs/core/tools/linker/backend/closure/ClosureLinkerBackend.scala

```diff
@@ -74,15 +74,14 @@ final class ClosureLinkerBackend(
     val builder = new ClosureAstBuilder(config.relativizeSourceMapBase)
 
     // Build Closure IR
-    logger.time("Emitter (create Closure trees)") {
-      emitter.emit(unit, builder, logger)
+    val coreJSLib = logger.time("Emitter (create Closure trees)") {
+      emitter.emitForClosure(unit, builder, logger)
```

Comment:
Ok. So let's leave it like this in this PR. But at the very least, I'd like to have a single abstraction for return values. Currently we have a builder and a return value, that's not very clean. But this can be cleaned up later indeed.